server/src/main/java/com/netcracker/domain/Message.java

```diff
@@ -0,0 +1,24 @@
+package com.netcracker.domain;
+
+public class Message {
+    private final MessageType messageType;
+    private final String data;
+
+    public Message(MessageType messageType) {
+        this.messageType = messageType;
+        data = "";
```

Comment:
I thought about using Command interface and I don't think that this is a good idea in my case, bacause I don't have big "If Block". Anyway, I can rewrite this class using Command interface when I will try to improve my code or have big "If Block".